isis/tests/BundleSettingsTests.cpp

```diff
@@ -0,0 +1,17 @@
+#include "BundleSettings.h"
+
+#include <gtest/gtest.h>
+
+class BoolTest : public ::testing::TestWithParam<bool> {
+  // Intentionally empty
+};
```

Comment:
This class is usable anywhere that we want to parameterize a test with a boolean. We probably will have a handful of these short classes that we'll re-use and keep in some kind of header.